recipe/meta.yaml

```diff
@@ -18,11 +18,19 @@ build:
 requirements:
   host:
     - python
+    - asteval>=0.9.11
+    - numpy
     - pip
+    - scipy>=0.17
+    - six
+    - uncertainties>=3.0
```

Comment:
Will it build without the added dependencies in the `host` section?

Reply:
I will test it.

Reply:
Thank you.